src/askui/models/router.py

```diff
@@ -6,21 +7,78 @@
 from ..utils import AutomationError
 from .ui_tars_ep.ui_tars_api import UITarsAPIHandler
 from .anthropic.claude_agent import ClaudeComputerAgent
+from abc import ABC, abstractmethod
 
+POINT = tuple[int, int]
```

Comment:
Like for introducing a type

Would it be fine from your side to use all uppercase only for constants and use PascalCase for types?

```suggestion
Point = tuple[int, int]
```